tests/integration/utils.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2022 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
```

Comment:
It might be worth integrating this back into the package itself or even aerial with some modifications.

Reply:
What do you think @ejfitzgerald

Reply:
It looks like 4757230328f8d36aa0c76b818cdc30054c57e0c2 copies it into the the actual API and updates the refrences. Can we remove this module, then?

Reply:
done

Reply:
@bryanchriswhite file actually removed  completely

Reply:
Sorry @solarw, I'm not sure why I was still seeing it earlier.